django_mongodb/compiler.py

```diff
@@ -490,6 +491,36 @@ def _get_ordering(self):
     def get_where(self):
         return self.where
 
+    def explain_query(self):
+        # Validate format (none supported) and options.
+        options = self.connection.ops.explain_query_prefix(
+            self.query.explain_info.format,
+            **self.query.explain_info.options,
+        )
+        # Build the query pipeline.
+        self.pre_sql_setup()
+        columns = self.get_columns()
+        query = self.build_query(
+            # Avoid $project (columns=None) if unneeded.
+            columns if self.query.annotations or not self.query.default_cols else None
+        )
+        pipeline = query.get_pipeline()
+        # Explain the pipeline.
+        kwargs = {}
+        if verbosity := options.get("verbosity"):
+            kwargs["verbosity"] = verbosity
```

Comment:
kwargs will only have a field or zero. could verbosity be passed in a different way to database.command?

Reply:
An alternative would be `verbosity = options.get("verbosity")` and change the `**kwargs` line to `**{"verbosity": verbosity} if verbosity else {}`.

The current approach seems more extensible. For example, there's a [`comment` kwarg](https://www.mongodb.com/docs/manual/reference/command/explain/) that I didn't implement.

Reply:
It was trivial to support `comment` by adding a loop.